docs/developers/contributing.md

```diff
@@ -23,7 +23,14 @@ Set the `upstream` remote to the base `napari` repository:
 git remote add upstream https://github.com/napari/napari.git
 ```
 
-Install the package in editable mode, along with all of the developer tools
+Napari can be installed on most macOS, Linux, and Windows systems with Python 3.8, 3.9, and 3.10 using pip.  First, 
+create a clean virtual environment:
+```python
+conda create -y -n napari-env -c conda-forge python=3.9
+conda activate napari-env
+```
+
+Install the package in editable mode, along with all of the developer tools:
```

Comment:
Tho create virtualenv one uses comment `python -m venv`. Here is the instruction of creating a fresh conda **environment** ent this instruction requires conda installed on a contributor machine. Conda is not present by default on any popular system. 

So it lacks, for example, the link to conda installation guide https://docs.conda.io/projects/conda/en/latest/user-guide/install/. 

There should also be information that python for python.org also works and a contributor could use plain python virtual environments or environments managing systems like `pyenv`

Reply:
That's a good point.  

That description was from:  https://napari.org/tutorials/fundamentals/installation.html?highlight=conda%20forge

@DragaDoncila Instead of including the description here, could the discussion just point the user to the installation.html page for details on how to install the environment?

Reply:
I think that would be better, yes. I'm also not sure about writing out the compatible Python versions, as they change reasonably often and it'll just be harder to keep it up to date. (And I would use 3.10 in the example just to extend its lifespan! 😂)

Reply:
Maybe we should put here badge (this one used in readme) and have python version information for free?

Reply:
> could the discussion just point the user to the installation.html page for details on how to install the environment?

Yeah I thought about this too, I think I just have a passionate dislike for guides that link me to other guides rather than just providing me with complete information. Still I think you're right - we shouldn't duplicate the information.

I'll look into whether the docs tooling supports any kind of `include` directive so we can keep snippets like this in one place but render them in multiple. That's outside the scope of this PR though imo